src/ostorlab/serve_app/types.py

```diff
@@ -59,11 +60,18 @@ class OxoKnowledgeBaseVulnerabilityType(graphene.ObjectType):
     recommendation = graphene.String()
 
 
+OxoRiskRatingEnum = graphene.Enum(
+    "OxoRiskRating",
+    [(risk.name.upper(), i) for i, risk in enumerate(utils_rik_rating.RiskRating)],
+)
+
+
 class OxoVulnerabilityType(graphene_sqlalchemy.SQLAlchemyObjectType):
     """SQLAlchemy object type for a vulnerability."""
 
     detail = graphene.Field(OxoKnowledgeBaseVulnerabilityType, required=False)
     cvss_v3_base_score = graphene.Float(required=False)
+    risk_rating = graphene.Field(OxoRiskRatingEnum, required=False)
```

Comment:
```suggestion
    risk_rating = graphene.Field(OxoRiskRatingEnum)
```
It's False by default

Reply:
explicit!!